debian/changelog

```diff
@@ -0,0 +1,5 @@
+quicktile (0.4.0gtk3-1) UNRELEASED; urgency=medium
+
+  * First Debian package, based on gtk3_port branch.
```

Comment:
Please remove the "based on gtk3_port branch" part. My intent is to merge `gtk3_port` into `master` and retire that name (in favour of having the one commit specific to the old `master` becoming a `legacy_gtk2` branch) either immediately before or immediately after merging this into it.

Reply:
Yes, sure will adjust this when merging into master. Just keeping it now because the builds I do are indeed "based on gtk3_port branch".

Reply:
In that case, maybe I should merge `gtk3_port` into `master`, let you revise this PR, and then merge it.

For simplicity's sake, I'd prefer to have it fixed before I merge it.

Reply:
Ok, will be done in a few minutes.